bower.json

```diff
@@ -3,5 +3,10 @@
   "main": [
     "src/js.cookie.js"
   ],
-  "ignore": []
+  "ignore": [
+    "travis.sh",
+    "test",
```

Comment:
We need this folder to be able to [integrate the encoding tests for server-side frameworks](https://github.com/js-cookie/js-cookie/blob/v2.0.3/CONTRIBUTING.md#integration-with-server-side).